pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -121,3 +121,14 @@ def test_to_excel_with_openpyxl_engine(ext, tmpdir):
 
     assert filename.exists()
     os.remove(filename)
+
+
+def test_to_excel_with_column_render(ext, tmpdir):
```

Comment:
is this not covered by the other writer tests?

Reply:
Sorry, I forgot that I added a test there too. I deleted this one.